src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -407,6 +409,28 @@ public function fileExists(string $path, $trackContents = true): bool
         return $exists;
     }
 
+    final public function willBeAvailable(string $package, string $class, array $parentPackages): bool
```

Comment:
this deserves some phpdoc explaining the goal of the method IMO

Reply:
docblock added